### PR TITLE
Adding the html and CSS files for the User Profile Page

### LINKED ISSUE #3: Complete HTML/CSS mock: User Profile

### PR DESCRIPTION
**Pull request by leah-mayo**
Addresses #4

### CODE REVIEW

**Comment by leah-mayo** [comment]
This was just a test pull request to see if everything was working. Once I actually push the mock then it can be reviewed.

**Comment by sperzion** [comment]
Should these be under a specific directory structure instead of at the root of the project directory?

**Comment by leah-mayo** [comment]
Yes, I merged my branch with Gwyn's directory-structure branch and moved my files to the webapp directory

File: user-profile-style.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.profile-header {
+
+}
+
+.profile-header__picture {
```
Comment (Google-JM) [inline user-profile-style.css:5]: Is "__" part of the style guide for css class naming? This is the first time I am seeing this style of naming.

**Review by Google-JM — APPROVED** [review]

**Review by sperzion — APPROVED** [review]

File: user-profile-style.css (hunk 1 above)
Comment (gwynethsc) [reply user-profile-style.css:5]: +1 Is there a particular reason for the double underscore?

Comment (leah-mayo) [reply user-profile-style.css:5]: I went by this style for naming, somehow it helped me to keep track of things better. But if I need to change it, I will.
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

**Pull request merged**